README.md

```diff
@@ -1,44 +1,69 @@
 # sapper-template
 
-The default [Sapper](https://github.com/sveltejs/sapper) template, available for Rollup and webpack.
+The default template for setting up a [Sapper](https://github.com/sveltejs/sapper) project. Can use either either Rollup or webpack as bundler.
 
 
 ## Getting started
 
 
 ### Using `degit`
 
-[`degit`](https://github.com/Rich-Harris/degit) is a scaffolding tool that lets you create a directory from a branch in a repository. Use either the `rollup` or `webpack` branch in `sapper-template`:
+To create a new Sapper project based on Rollup locally, run
 
 ```bash
-# for Rollup
 npx degit "sveltejs/sapper-template#rollup" my-app
-# for webpack
+```
+
+For a webpack-based project, instead run
+
+```bash
 npx degit "sveltejs/sapper-template#webpack" my-app
 ```
 
+[`degit`](https://github.com/Rich-Harris/degit) is a scaffolding tool that lets you create a directory from a branch in a repository.
+
+Replace `my-app` with the path where you wish to create the project.
+
 
 ### Using GitHub templates
 
-Alternatively, you can use GitHub's template feature with the [sapper-template-rollup](https://github.com/sveltejs/sapper-template-rollup) or [sapper-template-webpack](https://github.com/sveltejs/sapper-template-webpack) repositories.
+Alternatively, you can create the new project as a GitHub reposity using GitHub's template feature.
+
+Go to either [sapper-template-rollup](https://github.com/sveltejs/sapper-template-rollup) or [sapper-template-webpack](https://github.com/sveltejs/sapper-template-webpack) and click on "Use this template" to create a new project repository initialized by the template.
 
 
 ### Running the project
 
-However you get the code, you can install dependencies and run the project in development mode with:
+Once you have created the project, install dependencies and run the project in development mode:
 
 ```bash
 cd my-app
 npm install # or yarn
 npm run dev
 ```
 
-Open up [localhost:3000](http://localhost:3000) and start clicking around.
+This will start the development server on [localhost:3000](http://localhost:3000). Open it and click around.
 
-Consult [sapper.svelte.dev](https://sapper.svelte.dev) for help getting started.
+You now have a fully functional Sapper project! To get started developing, consult [sapper.svelte.dev](https://sapper.svelte.dev).
+
+### Using TypeScript
+
+By default, the template uses plain JavaScript. If you wish to use TypeScript instead, you need some changes to the project:
+
+ * Add `typescript` as well as typings as dependences in `package.json`
+ * Configure the bundler to use [`svelte-preprocess`](https://github.com/sveltejs/svelte-preprocess) and transpile the TypeScript code.
+ * Add a `tsconfig.json` file
+ * Update the project code to TypeScript
+
+The template comes with a script that will perform these changes for you by running
+
+```bash
+node scripts/setupTypeScriptRollup.js
```

Comment:
Conduitry suggested we drop `Rollup` from the name of this script and only create it for Rollup projects by modifying the scripts in https://github.com/sveltejs/sapper-template/tree/master/_template

Reply:
I saw that; I can create a new issue for it.